src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
```

Comment:
The class Figure should not be abstract if it's only used to set behavior for other classes. Consider using a regular class or an interface if you're not providing any default implementations for the methods in this class.